src/molecules/formfields/CreditCardField/Readme.md

```diff
@@ -1,5 +1,27 @@
 CreditCardField Example:
 
 ```example
-    <CreditCardField />
+    <CreditCardField>
+      <CreditCardField.Top>
+        <TextField
+          placeholder='Credit Card Number'
+          noBaseStyle
+        />
+      </CreditCardField.Top>
+
+      <CreditCardField.Bottom>
+        <TextField
+          placeholder='Expiration Date'
+          input={{
+            pattern: '[0-9]{13,16}'
```

Comment:
probably a question for @patrickkim, but I agree that we probably don't need it with redux-form validations so I'll remove.